mockwebserver-junit5/src/main/kotlin/mockwebserver3/junit5/internal/MockWebServerInstance.kt

```diff
@@ -15,20 +15,6 @@
  */
 package mockwebserver3.junit5.internal
 
-import mockwebserver3.MockWebServer
-import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.Test
-
-class InjectedConstructorTest(
-  private val server: MockWebServer
-) {
-  @Test
-  fun testOne() {
-    assertThat(server.started).isTrue()
-  }
-
-  @Test
-  fun testTwo() {
-    assertThat(server.started).isTrue()
-  }
-}
+annotation class MockWebServerInstance(
```

Comment:
Is this a JUnit best practice? I don’t love having to define a metaprogramming API here

Reply:
I think so, but I can switch to Named?

https://www.vogella.com/tutorials/JUnit5Extensions/article.html